kernel/safe_typing.mli

```diff
@@ -47,7 +47,7 @@ type private_constant_role =
   | Schema of inductive * string
 
 val side_effects_of_private_constants :
-  private_constants -> Entries.side_effects
+  private_constants -> Entries.side_effect list
```

Comment:
It used to be the "logic order" i.e. the order in which they had to be abstracted